pcap-dos.c

```diff
@@ -741,7 +741,7 @@ open_driver (const char *dev_name, char *ebuf, int promisc)
 }
 
 /*
- * Deinitialise MAC driver.
+ * Deinitialize MAC driver.
```

Comment:
Another British vs. US spelling issue.

Reply:
The US spelling is dominant: https://github.com/search?q=repo%3Athe-tcpdump-group%2Flibpcap%20initialize&type=code ...

When things aren't, I don't make such changes, but this repository is really heavily weighted to en-US, to the point where the single instances are effectively misspellings.

Reply:
$ git grep -i initialize|wc -l
198
$ git grep -i initialise|wc -l
3

Reply:
```
pcap-dos.c:666: * Initialise a named network device.
pcap-dos.c:853:   * set in it's probe handler), initialise near-memory ring-buffer for
pcap-dos.c:1004:   * just pretend Watt-32 is initialised okay.
```

Reply:
They sould be initialize(d).

Reply:
https://github.com/the-tcpdump-group/libpcap/compare/198fb737729d56610f85472fac1bca40b76a3aaf..295e0c5ab91525ebf2e7a7e4e6785c91316a1348